packages/mercury/docs/installation-for-angular.md

```diff
@@ -0,0 +1,377 @@
+│           ├── 📁 css
+│           ├── 📁 fonts
+│           └── 📁 icons
+```
+
+<details>
+<summary>Code explained:</summary>
+<br>
+Here we are defining tasks to copy the assets (css bundles, fonts, and icons) to the distribution folder of your app.
+
+</details>
+
+---
+
+## 4. Setup
+
+### 4.1. Import declarations
+
+Include the required imports at the application's entry point to properly configure and bootstrap the Mercury with Chameleon _(On the following code, we commented Angular declarations only to focus on Mercury and Chameleon related definitions)_:
+
+`src/main.ts` (Suggested file)
+
+```ts
+// import { bootstrapApplication } from "@angular/platform-browser";
+// import { appConfig } from "./app/app.config";
+// import { App } from "./app/app";
+
+import { defineCustomElements } from "@genexus/chameleon-controls-library/loader";
+import { registryProperty } from "@genexus/chameleon-controls-library/dist/collection";
+
+import { getImagePathCallbackDefinitions } from "@genexus/mercury/assets-manager.js";
+import { registerMercury } from "@genexus/mercury/register-mercury.js";
+import { setBundleMapping } from "@genexus/mercury/bundles.js";
+
+import { bundleToHashMappings } from "{{ CSS outDir path }}/bundle-to-hash-mappings";
+
+// Establishes the mapping between bundle names and their generated hashes.
+// For example, it maps the `components/button` bundle name to `button-e261832acea09e81.css`
+setBundleMapping(bundleToHashMappings);
+
+// It registers a mapping between the icons metadata and its path.
+registerMercury();
+
+// Register the default value for the getImagePathCallback property in all Chameleon
+// components. This implementation allows us to remove the need for binding the
+// getImagePathCallback property in all Chameleon controls that must render icons.
+registryProperty("getImagePathCallback", getImagePathCallbackDefinitions);
+
+// Setup the auto-loader for Chameleon components
+defineCustomElements(window);
+
+// Inject the base CSS bundle into the document head.
+// This bundle includes the design tokens needed to style all components.
+// This must be done in main.ts because the hashed filename is only available
+// via the bundleToHashMappings module, which cannot be used in index.html.
+const linkBaseBundle = document.createElement("link");
+linkBaseBundle.rel = "stylesheet";
+linkBaseBundle.href = `{{ CSS bundles final path }}${bundleToHashMappings["base/base"]}.css`;
+document.head.appendChild(linkBaseBundle);
+
+// (OPTIONAL - Only if you need to use icons)
+// Inject the base CSS bundle into the document head.
+// This must be done in main.ts because the hashed filename is only available
+// via the bundleToHashMappings module, which cannot be used in index.html.
+const linkIconsBundle = document.createElement("link");
+linkIconsBundle.rel = "stylesheet";
+linkIconsBundle.href = `{{ CSS bundles final path }}${bundleToHashMappings["base/icons"]}.css`;
+document.head.appendChild(linkIconsBundle);
```

Comment:
If this code is executed on the server, an error will occur. In other words, this won't work with SSR.

I suggest testing it first. If it errors, we can recommend removing the hash of the `base/base` and `base/icons` bundles (using the `avoid-hash` flag in the CLI) and then importing those files in the app.scss file that is used in the app.ts component.